Networking/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Networking",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Networking",
+            targets: ["Networking"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "Networking",
+            dependencies: []),
+        .testTarget(
+            name: "NetworkingTests",
+            dependencies: ["Networking"]),
+    ]
```

Comment:
Are you planning to add unit tests? If not, then please remove the test target and it's code

Reply:
I think, tests its a good practice